apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -5,24 +5,34 @@ import { Navbar } from '../../../components/Navbar.html.js';
 
 const examplePrompts = [
   {
-    id: 'random_mc',
-    question:
-      'Write a multiple choice question asking the user to choose the median of 5 random numbers between 1 and 100. Display all of the numbers to the user, and then make each of the numbers a potential answer, with the median being the correct answer.',
+    id: 'Select median of 5 random numbers',
+    promptGeneral:
+      'Write a multiple choice question asking the user to choose the median of 5 random numbers between 1 and 100. Display all numbers to the user, and ask them to choose the median.',
+    promptUserInput:
+      'Each random number generated should be a potential answer to the multiple-choice question. Randomize the order of the numbers.',
+    promptGrading: 'Make the actual median the correct answer.',
```

Comment:
Nit for consistency:

```suggestion
    promptGrading: 'The correct answer is the median of the numbers.',
```